tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/LiferayServerModuleLabelDecorator.java

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.server.ui;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.wst.server.core.IModule;
+import org.eclipse.wst.server.core.IServer;
+import org.eclipse.wst.server.core.IServerType;
+import org.eclipse.wst.server.ui.internal.ImageResource;
+import org.eclipse.wst.server.ui.internal.ModuleLabelDecorator;
+import org.eclipse.wst.server.ui.internal.view.servers.ModuleServer;
+
+/**
+ * @author Simon Jiang
+ */
+
+@SuppressWarnings( { "restriction" } )
+public class LiferayServerModuleLabelDecorator extends ModuleLabelDecorator
```

Comment:
I can't find the BundleDecorator class, it seems that you have been removed this class because I found you have been used this class in plugin.xml.